### DISCUSSION TITLE
ERR_SSL_VERSION_OR_CIPHER_MISMATCH

### DISCUSSION THREAD

**Discussion by ashab96** [vercel/community#2483] [Help] [answered]
### Summary

I  am using vercel as hosting provider for the first time, I deployed my project through github and configured  custom domain nocodecitizens.com.
when I visit the  url 
[https://www.nocodecitizens.com/](https://www.nocodecitizens.com/rl) or
[https://nocodecitizens.com/](https://nocodecitizens.com/) 
I am getting the error ERR_SSL_VERSION_OR_CIPHER_MISMATCH

<img width="791" alt="image" src="https://github.com/vercel/community/assets/28101438/7c5ac6ac-f245-4505-9c1d-349b697647a3">

if I visit the url  [http://nocodecitizens.com/](http://nocodecitizens.com/)  then it's default message as domain is parked with porkbun 
<img width="937" alt="image" src="https://github.com/vercel/community/assets/28101438/04538b30-bc39-4503-841a-a7a3f563fc4e">

The DNS records and SSL certificates configured in vercel are as below 
<img width="931" alt="image" src="https://github.com/vercel/community/assets/28101438/f29112a6-bb3e-4742-a2d7-9a3e843ef925">
<img width="929" alt="image" src="https://github.com/vercel/community/assets/28101438/ea75975e-fdc0-47ed-8355-0eebb019b921">

Deploy Message showing deployment to the custom domain 
<img width="701" alt="image" src="https://github.com/vercel/community/assets/28101438/fa7fcc2b-9d7e-4791-b1a3-7e71057c6d7a">

Domain Configuration in Vercel
<img width="743" alt="image" src="https://github.com/vercel/community/assets/28101438/92461535-22af-4e07-9557-b93d562847d8">

Vercel url Link of the project [https://front-notion-builder.vercel.app/](https://front-notion-builder.vercel.app/)

Please Advice me how can I resolve ERR_SSL_VERSION_OR_CIPHER_MISMATCH issue for https request and get the site deployed to http urls.

**Comment by Pieparker** [comment, marked as answer]
Hey there! Looks like the site is up and running. [The link mentioned](https://vercel.com/guides/why-is-my-vercel-domain-unverified#domain-is-verified-but-can't-access-the-website) in your screenshot will give you all of the information you need to understand what happened here!

In short, it may take a short while for DNS to propagate, and a few minutes for an SSL certificate to be issued.
